### ISSUE TITLE
Visual Studio Build: Cannot locate source files

### ISSUE THREAD

**Issue by JohnPeng47** [ctxis/capemon#9]
Cloned the repo and tried building the cuckoomon.sln file with Visual Studio 2017, using the platform v140 toolset. Ran into errors source files within the ./CAPE folder. Not sure why did is the case since it VS found the files in ./bson and ./distorm32-package just fine. 

![vs_include_dir](https://user-images.githubusercontent.com/9957837/45707564-2d5df700-bb0a-11e8-9dff-7f8ea12ef6c6.PNG)

**Comment by kevoreilly** [comment]
@redsand very kindly has updated capemon to work with VS2017 - please try merging his pull request: https://github.com/ctxis/capemon/pull/2

Let me know how it goes

**Comment by kevoreilly** [comment]
(The main branch is still VS2015 as my dev vms are running that - I'm in the process of making a shift to 2017)

**Comment by JohnPeng47** [comment]
Which version of Windows are you using @kevoreilly. I am getting the same "Cannot open source file error" using @redsand 's Visual Studio 2017 solution. I am running Windows 10

**Comment by doomedraven** [comment]
win7/winxp works fine

**Comment by JohnPeng47** [comment]
I've not used Visual Studio extensively before, so I'm not sure if this is normal behavior. The original error I had was that VS was not able to locate any of the source files in the CAPE folder.

![vs_include_dir](https://user-images.githubusercontent.com/9957837/45778406-7635ae80-bbe3-11e8-9b55-36e248185a76.PNG)

I then changed the folder name to all lower case "cape", updated the project .vcxproj  file, and the errors went away. However now it cannot locate any of the source files inside the "cape/Scylla" folder.

![screenshot 1](https://user-images.githubusercontent.com/9957837/45778774-708c9880-bbe4-11e8-8610-6c02632571ff.png)

Have any of you ran into something like this before?

**Comment by doomedraven** [comment]
nop, if you go to file open project and open solution, that shouldn't happen

**Comment by JohnPeng47** [comment]
That's what I did though. I think I'm gonna just switch over to Win7

**Comment by kevoreilly** [comment]
I haven't seen issues like that myself, but in the past have had issues with Visual Studio and files it misses or can't find - I have often found the answer lies in running Sysinternals Process Monitor to capture the relevant processes and see where VS is actually looking for stuff it can't find.

**Comment by doomedraven** [comment]
maybe jsut add include path in properties?

**Comment by JohnPeng47** [comment]
Ok so I've managed to build the monitor DLL's using @redsand's updated VS 2017 version. What I ended up doing was to change all the included directories to lowercase if they had any uppercase letters: ie. "CAPE" -> "cape" , " WTLInclude" -> "wtlinclude" , "Scylla" -> "scylla" and it was able to build without any problems. Me thinks this is probably a Win10 problem but either way, issue has been resolved.
